app/src/main/java/org/oppia/android/app/application/dev/DeveloperApplicationComponent.kt

```diff
@@ -86,7 +87,7 @@ import javax.inject.Singleton
     ExplorationStorageModule::class, DeveloperOptionsStarterModule::class,
     DeveloperOptionsModule::class, PlatformParameterSyncUpWorkerModule::class,
     NetworkConnectionUtilDebugModule::class, NetworkConfigProdModule::class, AssetModule::class,
-    LocaleProdModule::class, ActivityRecreatorProdModule::class,
+    LocaleProdModule::class, ActivityRecreatorProdModule::class, ActivityRouterModule::class,
```

Comment:
Is this new module also needed for the other application components (test, alpha, and Kenya-specific alpha)?

Reply:
Added `ActivityRouterModule` in other application components